core/src/main/scala/pureconfig/BasicConverters.scala

```diff
@@ -0,0 +1,106 @@
+package pureconfig
+
+import java.net.{ URI, URL }
+import java.nio.file.{ Path, Paths }
+import java.time._
+import java.util.UUID
+
+import scala.concurrent.duration.{ Duration, FiniteDuration }
+
+import com.typesafe.config._
+import pureconfig.error._
+
+/**
+ * Trait containing `ConfigConvert` instances for primitive types and simple classes in Java and Scala standard
+ * libraries.
+ */
+trait BasicConverters extends ConvertHelpers {
+
+  // primitive type converters
```

Comment:
I also do that on my on code, it's just that my threshold of "number of instances" for triggering that split is slightly higher :D But I understand the advantages and agree with the split.

Done in 866c95f.